README.md

```diff
@@ -1,6 +1,7 @@
 [🇨🇳](/README-cn.md "Simplified Chinese")
 [🇰🇷](/README-ko.md "Korean")
 [🇪🇸](/README-es.md "Spanish")
+[vn](/README-vn.md "Vietnamese")
```

Comment:
Can you use the 
🇻🇳 instead?

Reply:
Thank you for your feedback. I will use :vietnam: instead.
I am looking forward to somebody reviewing and giving good reviews for my contribution.